combat.md

```diff
@@ -1,3 +1,51 @@
 # Combat
 
+Combat in Blood Down the Gullet should feel mostly familiar to veterans of 4E. You'll roll initiative to determine the turn order for this combat. Within each turn, you'll have one move, one minor, and one standard action to take, with the ability to trade any down to a lesser action (taking an additional move instead of a standard, for example). You'll have a small library of ability cards as part of your character sheet, most of which will involve rolling a d20, adding some base score, and comparing the result to an enemy's defense value.
+
+The major differences that you'll notice will be in how skills are consumed and recharged, and how movement is tracked.
+
+## Skill Charges
+
+You still have **At-Will** abilities, which you can use as many times as you like. However, you'll no longer have encounter and daily powers. Instead, each non-At-Will ability will have a *charge number* on it.
+
+At the beginning of the day, put a number of charge tokens on each ability card corresponding to its charge number. Each time that you use one of those abilities, remove a charge token. Once the charge tokens are gone, you can't use the ability any more that day.
+
+Some abilities may give you the choice of using additional charge tokens to enhance their effect somehow. You have to choose to spend the tokens before you roll.
```

Comment:
These are basically psion augment points for everybody.

Mostly, I prefer this system because it encourages you to use *all* of your abilities. If you have encounter and daily powers, the tendency (or at least, I always do this) is to hoard all of your daily powers until you have some kind of reasonable confidence that you're in the "boss fight" or about to take a rest, then unload them all. By giving you some freedom to allocate your own power charges, and letting you supercharge some of them, I'm hoping to:

 1. Grant you a wider range of reasonable options in any particular combat round.
 2. Let you choose which of your attacks become awesome dramatic moments and which are your go-tos for filler rounds.
 3. Offer another resource that support roles can bolster.

Of course it might just be too much fiddling around with pennies. Like all of this, if it sucks we can change it!